src/templates/navigation/footer.pug

```diff
@@ -14,7 +14,7 @@ footer#footer-main.position-relative
               //-       path(d='M4 19.5A2.5 2.5 0 0 1 6.5 17H20')
               //-       path(d='M6.5 2H20v20H6.5A2.5 2.5 0 0 1 4 19.5v-15A2.5 2.5 0 0 1 6.5 2z')
               //-   span.btn-inner--text Documentation
-              a.btn.btn-github.my-2.ml-0.ml-sm-3(href='https://join.slack.com/t/necsi-edu/shared_invite/zt-d86ge6g0-sgLKgyhRpFBJq2VrnJOdhg', target='_blank') Join Us
+              a.btn.btn-github.my-2.ml-0.ml-sm-3(href='/volunteer') Join Us
```

Comment:
Please change to `router-link.btn.btn-github.my-2.ml-0.ml-sm-3(to='/volunteer') Join Us`